_posts/2021-03-23-activesupport.html

```diff
@@ -0,0 +1,172 @@
+---
+image: possessed-photography-tiNCpHudGrw-unsplash.jpg
+image_license: Photo by Possessed Photography on <a href="https://unsplash.com/photos/tiNCpHudGrw">Unsplash</a>
+title: ActiveSupport::Notifications is rad!
```

Comment:
https://capitalizemytitle.com/ to the rescue!!! :fire_engine:

```suggestion
title: ActiveSupport::Notifications is Rad!
```